src/System Application/App/User Settings/src/UserPersonalization.Page.al

```diff
@@ -116,6 +116,23 @@ page 9214 "User Personalization"
                             UserSettingsImpl.DisableTeachingTips(Rec."User SID");
                     end;
                 }
+                field("Legacy Action Bar"; LegacyActionBarEnabled)
+                {
+                    ApplicationArea = All;
+                    Caption = 'Legacy Action Bar';
+#pragma warning disable AA0219
+                    ToolTip = 'Revert the action bar to the legacy terms and layout. For example, the action bar won''t be pinned on pages, the Home tab is named Process, and split buttons become multiple actions, which means more clicks. Also, your options for personalization will be limited.';
```

Comment:
Same comment as for the table.

Reply:
This was given by Jacob, sticking with it.